schema/mysql/v8/visibility/schema.sql

```diff
@@ -135,9 +135,9 @@ CREATE INDEX by_keyword_07        ON custom_search_attributes (namespace_id, Key
 CREATE INDEX by_keyword_08        ON custom_search_attributes (namespace_id, Keyword08);
 CREATE INDEX by_keyword_09        ON custom_search_attributes (namespace_id, Keyword09);
 CREATE INDEX by_keyword_10        ON custom_search_attributes (namespace_id, Keyword10);
-CREATE FULLTEXT INDEX by_text_01  ON custom_search_attributes (namespace_id, Text01);
```

Comment:
Oh, now I understand why you added it here. Yes, apparently it won't work for FTS fields.